docs/content/faq/how-do-i-use-the-postgres-appliance.md

```diff
@@ -0,0 +1,18 @@
+---
+title: How do I use the Postgres appliance?
+layout: docs
+toc_min_level: 2
+---
+
+# How do I use the Postgres appliance?
+
+Flynn includes a highly available Postgres appliance which it uses for internal services and is available for use by installed apps.
+
+When adding a Postgres resource to an app, Flynn will provision a database and login credentials specifically for that app. That configuration is automatically passed to the app via environment variables:
```

Comment:
We should link to [Adding a database to an app](https://flynn.io/docs/postgres#adding-a-database-to-an-app) here